collections/Douce/MS_Douce_18.xml

```diff
@@ -37,43 +36,411 @@
+                     <decoNote type="decInit"> Further miniatures were apparently intended for large spaces in item 6. Next to the blank space preceding ‘Gloria in excelsis deo’ (fol. 35r) is the marginal instruction to the artist ‘Laus an[gelorum]’; presumably the Annunciation to the Shepherds was intended. In the margins next to preceding spaces, each accompanying a gospel reading, are three names of evangelists.</decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1900" notBefore="1800">
+                        <p>Brown leather over pasteboard, 19th century. Gilt border round the outer edge of both covers. Four raised bands on spine; gilt borders framing the raised bands and panels between them. Gilt lettering on spine: ‘MISSALE || MSS || SÆC. XV.’.</p>
+                        <p>Evidence of the former (presumably Bodleian) paper label on spine. Double gilt fillets on turn-ins. Gilt edges of textblock. Marks from the clasps of an earlier binding on fols. 258–259 and ii (?).</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
                      <origDate calendar="Gregorian" when="1433">c. 1433</origDate> and <origDate calendar="Gregorian" when="1475">c. 1475</origDate> (?) <origPlace>
                         <country key="place_7002445">English</country>
                      </origPlace>
+                     <p>Made perhaps for an Augustinian canon (depicted on fols. 68r and 159r). The motto (?) ‘suffre’ is written once in gold and once in silver next to his second ‘portrait’ (fol. 159r, psalm 101), and in plain ink (fol. 226r) at the end of a set of prayers. In the litany on fol. 219r ‘suffre’ is written in gold against the name of St Lawrence, and ‘me merci’ against the name of St Alban.</p>
                   </origin>
-                  <provenance notAfter="1834" notBefore="1757" resp="#MMM">
-                     <persName key="person_69848690" role="fmo">Francis Douce, 1757–1834</persName>
-                  </provenance>
-                  <acquisition resp="#MMM" when="1834">Bequeathed to the Bodleian in 1834</acquisition>
+                  <provenance>Masses and other texts added c. 1475 (Easter table for 1475–1481 on fol. 66v).</provenance>
+                  <provenance notAfter="1779" notBefore="1700">In the 18th century belonged to ‘S. Cooke’ (fols. 1v, 259v).</provenance>
+                  <provenance notAfter="1779" notBefore="1750">John Cooke, chaplain of Christ Church, Oxford (fol. ii recto); given in 1779 to:</provenance>
+                  <provenance when="1779">John Ireland, ‘pharmacopolæ’ (fol. ii recto).</provenance>
+                  <provenance notAfter="1834" notBefore="1757">
```

Comment:
Both owners in lines 434-5 should have authority files created: I wonder if John Ireland apothecary is this person http://www.oxfordhistory.org.uk/doctors/apothecaries/ireland_john.html  John Cooke seems to be CCED https://theclergydatabase.org.uk/jsp/persons/CreatePersonFrames.jsp?PersonID=12322 which is the same person as http://viaf.org/viaf/1063159233907303370153/#Cooke,_John,_fl._1783

Reply:
Well spotted!